solc/CommandLineInterface.cpp

```diff
@@ -63,20 +63,20 @@
 #include <boost/range/adaptor/filtered.hpp>
 #include <boost/algorithm/string.hpp>
 
-#ifdef _WIN32 // windows
-	#include <io.h>
-	#define isatty _isatty
-	#define fileno _fileno
-#else // unix
-	#include <unistd.h>
+#ifdef _WIN32  // windows
+#include <io.h>
+#define isatty _isatty
+#define fileno _fileno
+#else  // unix
+#include <unistd.h>
```

Comment:
It removes indents from nested pragmas. Technically they're supposed to start at the first column but do we really want to do it that way? It's totally unreadable.